packages/alfa-dom/src/node/traversal/get-nodes-between.ts

```diff
@@ -62,7 +63,11 @@ export function getNodesBetween(
 /**
  * Get all nodes between node1 and node2, included.
  */
-function getNodesInclusivelyBetween(node1: Node, node2: Node): Sequence<Node> {
+function getNodesInclusivelyBetween(
+  node1: Node,
+  node2: Node,
+  treeOptions: Node.Traversal
```

Comment:
Should this also have the default value `{ flattened: true, nested: true }`?

Reply:
This is not needed because that function is not exported and only called from the "parent" `getNodesBetween`. If we need to promote it, it should have a default, but that shouldn't be necessarily given that the `includeOptions` of the "parent" would cover the functionality…